mass_editing/wizard/mass_editing_wizard.py

```diff
@@ -20,194 +20,256 @@
 #
 ##############################################################################
```

Comment:
Please change to new OCA header https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py#L3